src/lib/langchain/prompts/changelog.ts

```diff
@@ -0,0 +1,17 @@
+import { PromptTemplate } from 'langchain/prompts'
+
+const template = `Write informative git changelog, in the imperative, based on a series of individual messages.
+
+- Typically a hyphen or asterisk is used for the bullet
+- Summarize dependency updates
+
+"""{summary}"""
+
+Changelog:`
+
+export const inputVariables = ['summary']
+
+export const CHANGELOG_PROMPT = new PromptTemplate({
+  template,
+  inputVariables,
+})
```

Comment:
Templates could be enhanced later on to use markdown instead of raw text.  Typically LLMs respond better to text formatted in markdown than raw text as well.